src/cclib/io/moldenwriter.py

```diff
@@ -10,19 +10,28 @@
 import os.path
 
 from . import filewriter
+from .filewriter import MissingAttributeError
 from cclib.parser import utils
 
 
 class MOLDEN(filewriter.Writer):
     """A writer for MOLDEN files."""
 
+    def _check_required_attributes(self, required):
```

Comment:
Yes, this would be more useful. I'll make the change. Though I don't think it would break anything.